service/src/main/java/bio/terra/pipelines/dependencies/sam/SamService.java

```diff
@@ -65,15 +65,15 @@ public ApiSystemStatusSystems checkHealthApiSystemStatus() {
         .addMessagesItem(healthResult.message());
   }
 
-  public String getTspsServiceAccountToken() {
+  public String getTeaspoonsServiceAccountToken() {
     try {
       GoogleCredentials creds =
           GoogleCredentials.getApplicationDefault().createScoped(SAM_OAUTH_SCOPES);
       creds.refreshIfExpired();
       return creds.getAccessToken().getTokenValue();
     } catch (IOException e) {
       throw new InternalServerErrorException(
-          "Internal server error retrieving TSPS credentials", e);
+          "Internal server error retrieving Teaspoons credentials", e);
```

Comment:
since this could get returned to the user, what if we just say "Internal server error retrieving service credentials"

Reply:
👍